homeassistant/components/fritzbox_callmonitor/config_flow.py

```diff
@@ -119,7 +119,7 @@ async def _get_name_of_phonebook(self, phonebook_id):
         phonebook_info = await self.hass.async_add_executor_job(
             self._fritzbox_phonebook.fph.phonebook_info, phonebook_id
         )
-        return phonebook_info[FRITZ_ATTR_NAME]
+        return phonebook_info[ATTR_NAME]
```

Comment:
It's important that we don't use Home Assistant common constants with external APIs. This looks like it's a fritzbox api we're accessing.

Reply:
Ok, got that. I'll open a PR to roll that back.

Reply:
PR for rollback: https://github.com/home-assistant/core/pull/47211